docs/develop/01_sdk/02_cookbook/04_claiming/01_ctype_creation.md

```diff
@@ -8,35 +8,39 @@ import TsJsBlock from '@site/src/components/TsJsBlock';
 import CreateCType from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/claiming/01_create_ctype.ts';
 import FetchCType from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/claiming/02_fetch_ctype.ts';
 
+Every KILT credential has to conform to a CType.
+A CType describes which properties a credential has and what type these properties have.
+CTypes must be registered on the Spiritnet blockchain.
+To learn more about the CTypes, see the [CType concept section](../../../../concepts/05_credentials/02_ctypes.md)
+
 The creation of a CType in KILT involves two steps: the definition of a CType and the anchoring of its hash on the KILT blockchain.
 
-:::info
+:::info DID required
 The creator of a CType is required to have a full DID with an attestation key.
 To see how to manage DIDs, please refer to the [DID section](../01_dids/03_full_did_update.md).
 :::
 
+:::info CTypes are unique
+The creation of a new CType requires the CType hash to be unique.
+Before writing a new CType, Attesters should check whether there is already an existing CType which matches their requirements.
+
+Visit our [CType index repository](https://github.com/KILTprotocol/ctype-index) for a non-exhaustive list of existing CTypes.
+:::
+
 The following snippets show how to create a CType:
 
 <TsJsBlock>
   {CreateCType}
 </TsJsBlock>
 
-:::note
-The creation of a new CType requires the CType hash to be unique.
-Before writing a new CType, Attesters should check whether there is already an existing CType which matches their requirements.
-Visit our [CType index repository](https://github.com/KILTprotocol/ctype-index) for a non-exhaustive list of existing CTypes.
-:::
 
 ## Retrieve a CType from its ID
 
-As of the [KILT runtime 1.9.0][kilt-runtime-1.9.0], CTypes can be queried directly from any KILT archive nodes.
-
-This is possible because the creation of a new CType now writes on chain its creation block number, that can be used as a "pointer" to query the full block content, including its list of transactions.
+CTypes can be queried directly from any KILT archive nodes.
+This is possible because the CType is written onchain when they are registered.
```

Comment:
At this point, I would simply remove this line here, since it's clear that it's written on chain when it's registered [on chain].

Reply:
[03c4bea](https://github.com/KILTprotocol/docs/pull/244/commits/03c4bea55ac4c369e9f612200aa4b4f713d5e3c8)